Extension/i18n/csy/walkthrough/installcompiler/install-compiler-windows.md.i18n.json

```diff
@@ -11,12 +11,12 @@
 	"walkthrough.windows.text2": "Pokud chcete nainstalovat MSVC, stáhněte {0} ze stránky Visual Studio {1}.",
 	"walkthrough.windows.build.tools1": "Build Tools for Visual Studio 2022",
 	"walkthrough.windows.link.downloads": "Položky ke stažení",
-	"walkthrough.windows.text3": "V Instalační program pro Visual Studio zkontrolujte {0} úlohy a vyberte {1}.",
+	"walkthrough.windows.text3": "V Instalačním programu pro Visual Studio zkontrolujte úlohy {0} a vyberte {1}.",
 	"walkthrough.windows.build.tools2": "C++ Build Tools",
 	"walkthrough.windows.link.install": "Nainstalovat",
 	"walkthrough.windows.note1": "Poznámka",
 	"walkthrough.windows.note1.text": "Můžete použít sadu nástrojů C++ z Visual Studio Build Tools spolu s Visual Studio Code ke kompilaci, sestavení a ověření jakékoli kódové báze C++, pokud máte také platnou licenci Visual Studio (buď Community, Pro nebo Enterprise), kterou aktivně používáte k vývoji kódové báze C++.",
-	"walkthrough.windows.open.command.prompt": "Otevřete ho {0} zadáním „{1}“ v nabídce Start ve Windows.",
+	"walkthrough.windows.open.command.prompt": "Otevřete ho {0} zadáním '{1}' v nabídce Start ve Windows.",
```

Comment:
When we want to control the type of quotes used, I recommend including them in the substitution string and removing them from here where the translators can change them.

Reply:
@bobbrow The type of quote doesn't need to controlled here does it?

Reply:
Is this coming from the localizers then? IOW, it's not your change?

Reply:
It's not my change -- it's from the loc team. In this PR, I didn't make any major changes to any strings. You can confirm that by looking at the unaltered localization branch at https://github.com/microsoft/vscode-cpptools/pull/13181 or doing a diff of the branches. I just made fixes for the issues which all have bugs filed on them currently, some new as of today, some re-activated today.

Reply:
Different languages have different quote preferences so I assume the quote preference is at intended, although it might not be, I haven't looked into it.